packages/arb-token-bridge-ui/tests/e2e/specs/redeemRetryable.cy.ts

```diff
@@ -5,6 +5,7 @@ import {
   getInitialERC20Balance,
   getL2NetworkConfig
 } from '../../support/common'
+import { normalizeTimestamp } from 'packages/arb-token-bridge-ui/src/state/app/utils'
```

Comment:
e2e doesn’t like import { normalizeTimestamp } from 'packages/arb-token-bridge-ui/src/state/app/utils'

sorry i guess Math.floor(Date.now() / 1000).toString() is fine